### PR TITLE
feat(dev-env): Pin yarn version

### PR DESCRIPTION
**Pull request by roninjin10**
Yarn is a small (or at least smaller than you'd expect) javascript file.  Committing it in the monorepo guarantees that anybody using the monorepo is all using the latest and consistent version of yarn.  Run `yarn update:yarn` to update the version.

This matches what we did in gateway in this pr https://github.com/ethereum-optimism/gateway/pull/417

### CODE REVIEW

**Comment by roninjin10** [comment]
Messed up git commits completely fixing

File: packages/common-ts/src/base-service/base-service-v2.ts [hunk 1]
```diff
@@ -34,9 +34,7 @@ export type OptionsSpec<TOptions extends Options> = {
   }
 }
 
-export type MetricsV2 = {
```
Comment (mslipper) [inline packages/common-ts/src/base-service/base-service-v2.ts:37]: Did you mean to include this?

File: .changeset/clever-bees-visit.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+'@eth-optimism/common-ts': minor
```
Comment (mslipper) [inline .changeset/clever-bees-visit.md:2]: Did you mean to include this?

**Review by mslipper — CHANGES_REQUESTED** [review]

**Comment by roninjin10** [comment]
@mslipper i messed up my git but fixed.   

Seperately, I included a changeset that minor bumps every package but that seems odd since I technically changed 0 packages here.

**Comment by mslipper** [comment]
You don't need that changeset - the Mergify comment is just a reminder 😄

**Review by tynes — APPROVED** [review]

**Review by mslipper — APPROVED** [review]

**Pull request merged**